packages/variable-resolver/src/browser/common-variable-contribution.ts

```diff
@@ -18,6 +18,7 @@ import { injectable, inject } from 'inversify';
 import { VariableContribution, VariableRegistry } from './variable';
 import { EnvVariablesServer } from '@theia/core/lib/common/env-variables';
 import { CommandService } from '@theia/core/lib/common/command';
+import { isWindows } from '@theia/core/lib/common/os';
```

Comment:
Out of curiosity, what's the intention here? So if the backend is a Linux machine and the frontend is running on a Windows node, what do you want to see: `\\` or `/`?

If you need the backend OS, you have to use:

https://github.com/eclipse-theia/theia/blob/f1d469033cfebb19d9e54215c9b746d133bd0206/packages/core/src/node/application-server.ts#L45-L47

If you were aware of all these 👆 , just ignore my comment. 😊

Reply:
@kittaakos 
I agree, good point. Updated.